utility/aws.py

```diff
@@ -238,3 +262,53 @@ def detach_and_delete_volume(self, volume, timeout=120):
             "Delete response for volume: %s is: %s", ec2_volume.volume_id,
             delete_response
         )
+
+    def stop_ec2_instance(self, instance_id, wait=False):
+        """
+        Stopping an instance
+
+        Args:
+            instance_id (str): ID of the instance to stop
+            wait (bool): True in case wait for status is needed,
+                False otherwise
+
+        Returns:
+            bool: True in case operation succeeded, False otherwise
+        """
+        res = self.ec2_client.stop_instances(
+            InstanceIds=[instance_id], Force=True
+        )
+        if wait:
+            for sample in TimeoutSampler(
+                TIMEOUT, SLEEP, self.get_instances_status_by_id, instance_id
+            ):
+                if sample == STOPPED:
+                    return True
+            return False
+        state = res.get('StoppingInstances')[0].get('CurrentState').get('Code')
+        return state == STOPPING
+
+    def start_ec2_instance(self, instance_id, wait=False):
+        """
+        Starting an instance
+
+        Args:
+            instance_id (str): ID of the instance to start
+            wait (bool): True in case wait for status is needed,
+                False otherwise
+
+        Returns:
+            bool: True in case operation succeeded, False otherwise
+        """
+        res = self.ec2_client.stop_instances(
```

Comment:
change to start_instances()
Is there a need for 'Force' arg in start?

Reply:
done